python/Ganga/Core/GangaRepository/VStreamer.py

```diff
@@ -361,7 +364,9 @@ def end_element(name):
                 # unescape the special characters
                 s = unescape(self.value_construct)
                 #logger.debug('string value: %s',s)
-                val = eval(s, config_scope)
+                if s not in _eval_strings:
+                    _eval_strings[s] = eval(s, getProxyInterface().__dict__)
+                val = copy.deepcopy(_eval_strings[s])
```

Comment:
yes, if the interface (```GPI``` or ```ganga```) didn't change... besides this has to eval an actual string not just find an object. We're doing some horrible, horrible things here fyi.